source/release-notes.txt

```diff
@@ -10,6 +10,34 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.40.3
+----------------
```

Comment:
(future work) 
What do you think about drafting a code comment in google doc providing some guidance about how we do updates in Compass so we can be consistent, release over release. Once done, you could add to the this page, right under `..contents::`. 

Main thing is tenses. We could also provide some guidance on whether or not to add "font treatment". 

The other thing I'd like to see is in the New Features section is links to whatever new docs are available.   Let me know if you're open to taking a stab at this. I'm also happy to do a first pass. Whatever you'd like.

Reply:
Thanks for taking this in stride Ian! This is a great approach and I really appreciate it!